cmd/fs-v1-helpers.go

```diff
@@ -228,15 +228,6 @@ func fsStatDir(ctx context.Context, statDir string) (os.FileInfo, error) {
 	return fi, nil
 }
 
-// Returns if the dirPath is a directory.
-func fsIsDir(ctx context.Context, dirPath string) bool {
```

Comment:
`getObjectInfo` function calls `fsIsDir`.

Reply:
It doesn't @ebozduman

Reply:
That has been removed, otherwise this PR won't build.